templates/hpe/index.html

```diff
@@ -1,448 +1,558 @@
 {% extends "hpe/_base_hpe.html" %}
```

Comment:
Large diff only due to linting, please see comments in below where I've flagged the minor changes